Core/Core/View/Base/AppReview/AppReviewView.swift

```diff
@@ -50,7 +50,7 @@ public struct AppReviewView: View {
                         .font(Theme.Fonts.titleMedium)
```

Comment:
@rnr  The title is using the default color, can you please also update it to use `textPrimary`?

Reply:
done